### PR TITLE
Added support for one-to-many relations

### PR DESCRIPTION
**Pull request by tkuska**
Now we can pass collection of entities to form field instead of single entity.

### CODE REVIEW

**Comment by Glifery** [comment]
Hi @tkuska.
Thanks for the work! We're preparing this repo for symfony 3 support. And it should be done first. So I'll review your PR and merge it asap.
Actually you can help us to prepare your PR to symfony 3. All you need to do is to merge a `symfony-3-support` branch into your PR's branch and make sure all works. Thanks!

**Comment by tkuska** [comment]
It looks like everything works fine after merge `symfony-3-support` branch

File: Form/DataTransformer/ObjectToIdTransformer.php [hunk 1]
```diff
@@ -10,8 +10,8 @@
 use Symfony\Component\Form\Exception\InvalidConfigurationException;
 use Symfony\Component\Form\Exception\TransformationFailedException;
 
-class ObjectToIdTransformer implements DataTransformerInterface
-{
+class ObjectToIdTransformer implements DataTransformerInterface {
```
Comment (Glifery) [inline Form/DataTransformer/ObjectToIdTransformer.php]: According to [4.1. Extends and Implements (PSR-2)](http://www.php-fig.org/psr/psr-2/#extends-and-implements)
> The opening brace for the class MUST go on its own line;

please fix this

File: Form/DataTransformer/ObjectToIdTransformer.php [hunk 2]
```diff
@@ -31,58 +31,87 @@ class ObjectToIdTransformer implements DataTransformerInterface
...
+    protected $multiple;
+
+    /**
      * @param ManagerRegistry $registry
      * @param string $class
      * @param string $property
      */
-    public function __construct(ManagerRegistry $registry, $om, $class, $property)
-    {
+    public function __construct(ManagerRegistry $registry, $om, $class, $property, $multiple = false) {
```
Comment (Glifery) [inline Form/DataTransformer/ObjectToIdTransformer.php]: According to [4.3. Methods (PSR-2)](http://www.php-fig.org/psr/psr-2/#methods)
> The opening brace MUST go on its own line,

please fix this
(note that there are a couple of places in code that should be fixed)

**Review by Glifery — CHANGES_REQUESTED** [review]
Hi @tkuska.
Great work! Now we can merge it.
By the way there are some codestype problems that should be fix. Please take a look my review

**Comment by tkuska** [comment]
fixed :)

**Comment by tkuska** [comment]
now it's fixed, i misunderstood your previous comment

**Pull request merged**